requirements/docs.txt

```diff
@@ -13,3 +13,6 @@ sphinx-copybutton>=0.3
 
 # integrations
 pytorch-lightning>=1.1
+
+# add extra requirements
+-r text.txt
```

Comment:
any reason why this needs to be added to the docs requirements?

Reply:
Adding this let the doctests pass.

Reply:
Adding this let the doctests pass.